phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace phpdotnet\phd;
 
+#[\AllowDynamicProperties]
```

Comment:
This is sad :(

Reply:
As I said above, there are some dynamic properties in these files. :-)  I'll make a note of this for a follow up PR.